.buildkite/test-pipeline.yaml

```diff
@@ -469,8 +480,6 @@ steps:
   - pytest models/encoder_decoder/vision_language/test_broadcast.py -v -s -m 'distributed(num_gpus=2)'
   - pytest models/decoder_only/vision_language/test_models.py -v -s -m 'distributed(num_gpus=2)'
   - pytest -v -s spec_decode/e2e/test_integration_dist_tp2.py
-  - pip install -e ./plugins/vllm_add_dummy_model
-  - pytest -v -s distributed/test_distributed_oot.py
```

Comment:
Although we probably can't move this because it needs distributed environment...

Reply:
Alternatively we can require 2 GPUs for the full Plugin Test pipeline.

Reply:
Oh, my mistake. Will update later to avoid wasting CI resource.